models/StudentRecord.go

```diff
@@ -39,6 +40,10 @@ func GetStudentRecords(grade uint32, subjectArray []string, mediumArray []string
 		tx = tx.Where("medium in (?)", mediumArray)
 	}
 
+	if language != "" {
```

Comment:
Is this necessary? language will definitely be passed right? So there isn't a case where ```language == ""```